pkg/util/cert/cert.go

```diff
@@ -90,7 +90,7 @@ func NewSignedCert(cfg Config, key *rsa.PrivateKey, caCert *x509.Certificate, ca
 	certTmpl := x509.Certificate{
 		Subject: pkix.Name{
 			CommonName:   cfg.CommonName,
-			Organization: caCert.Subject.Organization,
+			Organization: cfg.Organization,
```

Comment:
@liggitt Is this reasonable to do?
I guess we should look if somebody is using this code so we don't break anyone...